src/main/scala/cromwell/Main.scala

```diff
@@ -51,8 +48,18 @@ object Main extends App {
   }
 
   def highlight(args: Array[String]) {
+    if (args.length >= 2 && !Seq("html", "console").contains(args(1))) {
+      println("usage: highlight <wdl> <html|console>")
+      System.exit(1)
+    }
+
+    val visitor = args match {
+      case a if a.length >= 2 && args(1) == "html" => HtmlSyntaxHighlighter
```

Comment:
Fair point on the first bit about CLI. 

For my own particular use case I want it to be easy to turn off as I'd prefer it to always be off :) It'd be nice to throw that into my local config file that's always passed in anyways.

I suppose I could make a script or something to automate things like turning it off.